esinstall/package.json

```diff
@@ -56,5 +56,9 @@
     "rollup-plugin-node-polyfills": "^0.2.1",
     "validate-npm-package-name": "^3.0.0",
     "vm2": "^3.9.2"
+  },
+  "devDependencies": {
+    "@types/es-module-lexer": "^0.3.0",
```

Comment:
We actually want to use the CJS es-module-lexer interface, which has different behavior than the ESM interface that TypeScript supports. I believe that this is a no-op because we're requiring it, so safe to remove for now.

Reply:
ok, I found es-module-lexer interface in the top-level package.json

![image](https://user-images.githubusercontent.com/15937065/102693237-380d1680-4254-11eb-9b6e-c386f748b70d.png)

Reply:
oh, interesting! I may have gotten that wrong, and just forgotten that we manage our types in the top-level